includes/Elements/Section/Section.php

```diff
@@ -515,6 +533,33 @@ public function before_render( $options ) {
 	 * @return void
 	 */
 	public function render( $options ) {
+		$masks = $options->get_value( 'shapes' );
+
+		if ( ! empty( $masks ) ) {
+			foreach ( $masks as $key => $shape ) {
+				$shape_path   = $shape['shape'];
+				$shape_color  = ( ! empty( $shape['color'] ) ) ? sprintf( 'color: %s;', $shape['color'] ) : '';
+				$shape_height = ( ! empty( $shape['height'] && $shape['height'] !== 'auto' ) ) ? sprintf( ' height: %s;', $shape['height'] ) : '';
+				if ( ! empty( $shape_path ) ) {
+					$shape_style  = '';
+					$shape_style .= $shape_color;
+					$shape_style .= $shape_height; ?>
+					<span class="znpb-mask <?php echo '-pos--' . esc_attr( $key ); ?>"
```

Comment:
znpb is reserved for backend/editor css. For frontend please use the .zb prefix

Also, please use prefixed class names ( -pos-- should be prefixed). This is necessary in order to work with other themes/plugins that can use the same unprefixed CSS class name

Don't forget to change this to the vue component as well